scylla/src/transport/retry_policy.rs

```diff
@@ -299,7 +299,7 @@ mod tests {
                         (got 1 values, 2 statements)"
                 .to_owned(),
         )));
-        default_policy_assert_never_retries(QueryError::ProtocolError("test"));
+        default_policy_assert_never_retries(ProtocolError::RepreparedIdChanged.into());
```

Comment:
Why such choice of `ProtocolError` variant?

Reply:
No specific reason - we just want to test that DRP decides that we should not retry the query when protocol error occurred.

Reply:
And this is variant which can be easily constructed (without additional arguments)